src/java.base/share/classes/javax/net/ssl/ExtendedSSLSession.java

```diff
@@ -163,4 +164,103 @@ public List<SNIServerName> getRequestedServerNames() {
     public List<byte[]> getStatusResponses() {
         throw new UnsupportedOperationException();
     }
+
+    /**
+     * Generate Exported Keying Material (EKM) calculated according to the
```

Comment:
Suggest saying "Generates" (it is an action).

Reply:
Done.